lib/phlex/bucket.rb

```diff
@@ -1,24 +1,41 @@
 # frozen_string_literal: true
 
 module Phlex::Bucket
+	module Proxy
+		def method_missing(name, *args, **kwargs, &block)
+			if self.class.constants.include?(name) && self.class.const_get(name) && methods.include?(name)
+				public_send(name, *args, **kwargs, &block)
+			else
+				super
+			end
+		end
+
+		def respond_to_missing?(name, include_private = false)
+			self.class.constants.include?(name) && self.class.const_get(name) && methods.include?(name)
+		end
```

Comment:
I’m not 100% sure if we want these `respond_to_missing?` methods because they may slow down `respond_to?` calls.